### DISCUSSION TITLE
How can we ignore a dynamic value in a Not command. Its not accepting regex

### DISCUSSION THREAD

**Discussion by kishordevakumar** [cypress-io/cypress#20879] [General]
I need to ignore a value from a not condition. but its not accepting it. since it is dynamic

.not('Selected')

**Comment by jjhelguero** [comment]
Are you trying to use a regex or just filter the `:not(Selected)` out?

**Reply by kishordevakumar** [reply]
:not(Selected) it is not working , because the Selected is dynamic , it will change. I need to ignore that element selector

**Reply by jjhelguero** [reply]
If you could provide some DOM details or an example, that would be more helpful.
